cmake/ecbuild_get_test_data.cmake

```diff
@@ -386,7 +409,7 @@ endfunction(ecbuild_get_test_data)
 
 function( ecbuild_get_test_multidata )
 
-    set( options EXTRACT NOCHECK )
+    set( options EXTRACT NOCHECK INSECURE_CURL)
```

Comment:
Same as above, the name ought to be `INSECURE`

Reply:
Fixed.